ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/LotsOfPartiesIT.scala

```diff
@@ -16,24 +16,24 @@ final class LotsOfPartiesIT extends LedgerTestSuite {
     "LOPseeTransactionsInMultipleSinglePartySubscriptions",
     "Observers should see transactions in multiple single-party subscriptions",
     allocate(NoParties),
-  )(_ => _ => Future.failed(Retired))
+  )(_ => { case _ => Future.failed(Retired) })
```

Comment:
I failed to find a way to overload this to work for both `PartialFunction` and regular functions and on Scala 2.12 (but not 2.13), this otherwise results in a type error.